aptos-move/vm-genesis/src/lib.rs

```diff
@@ -408,6 +414,7 @@ pub struct Validator {
 
 pub struct TestValidator {
     pub key: Ed25519PrivateKey,
+    pub consensus_key: bls12381::PrivateKey,
```

Comment:
I'm a bit confused here: Why do we still have a `pub key: Ed25519PrivateKey` here? What would it be used for?

Reply:
that `key` is for the root account

Reply:
That deserves a better name, but it should probably be done in a different pull request...